source/cloud/cloud-user-management/cloud-groups.rst

```diff
@@ -129,11 +129,15 @@ Users who have logged in and accessed Mattermost will be visible in the members
 Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels unless the team or channel is group-synced.
 
 .. note::
-   When a member removes themselves manually from a channel, that action is tracked in the **Channel Member History** table.  Users are not re-added to channels from which they previously removed themselves.
+   When a member removes themselves manually from a channel, that action is tracked in the **Channel Member History** table. If a System Admin manually forces all members in a specific LDAP group to join the teams and channels synchronized to that group, members can potentially be re-added to channels from which they previously removed themselves.
```

Comment:
We're not tracking who performed the removal from the channel. I mention it in case you think users might get that impression from the use of "removes themselves".

Reply:
Yeah, I think we should change this to be "re-added to channels from which they were previously removed."

Reply:
Thank you both -- I've applied your excellent clarification to both the cloud and the self-managed notes.